apps/x509.c

```diff
@@ -567,8 +567,6 @@ int x509_main(int argc, char **argv)
             goto end;
         }
 
-        if (!X509_set_issuer_name(x, X509_REQ_get_subject_name(req)))
-            goto end;
```

Comment:
Keep this call as is, please, as in case `-CA` is not used there will be no issuer set.